crowbar_framework/spec/models/service_object_spec.rb

```diff
@@ -118,8 +118,8 @@
       it "does not allow clusters of nodes to be assigned" do
         dns_proposal.elements["dns-client"] = ["cluster:test"]
 
-        dns_service.stubs(:role_constraints).returns({ "dns-client" => { "cluster" => false, "admin" => true } })
-        dns_service.stubs(:is_cluster?).returns(true)
+        allow(dns_service).to receive(:role_constraints).and_return({ "dns-client" => { "cluster" => false, "admin" => true } })
```

Comment:
Metrics/LineLength: Line is too long. [128/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.